### DISCUSSION TITLE
Versioning

### DISCUSSION THREAD

**Discussion by mordilos** [ENHANCE-PET/MOOSE#145] [General]
Hello, i was just wondering about the versioning scheme. In the documentation the version is Moose 3.0 and in the tags the latest one is 2.2.37! What is the meaning behind Moose 3.0? Cheers!

**Comment by LalithShiyam** [comment]
Haha @mordilos hi there! 

I think it says coming soon - so we will release it this month. And I would wait till October 19th. We have the final models in training and we will release it for the start of the EANM. The pypi version is the right one. 

Cheers,
Lalith

**Reply by mordilos** [reply]
You are absolutely right! I missed the title. Anyways, thanks! 🙏

**Reply by LalithShiyam** [reply]
Will hit you up when everything is set - hopefully you will be pleasantly surprised by the upgrade!

**Comment by LalithShiyam** [comment]
Hi @mordilos moosev3 is out!

**Comment by mordilos** [comment]
Hi @LalithShiyam! I have tried it since day 1! Thanks for the heads up 🍻

**Reply by LalithShiyam** [reply]
@mordilos fantastic, hopefully you liked it!

**Reply by mordilos** [reply]
Yes, it is awesome!

**Comment by LalithShiyam** [comment]
great, closing this discussion then :D!
